tox.ini

```diff
@@ -42,3 +42,12 @@ commands =
     {env:TRAVIS_WAIT:} molecule idempotence -s source
     molecule side-effect -s source
     molecule verify -s source
+    molecule lint -s dynamic
```

Comment:
I recently fixed a bug in tox.ini; the "destroy" stage needs to be added for the "source" scenario before we run the "dynamic" scenario.

In other words, only the last scenario should lack a "destroy" task. (It wastes time, and makes debugging more difficult.) (Maybe we should add a comment for this.)

Reply:
+1 to add a comment